pyfar/classes/audio.py

```diff
@@ -205,14 +206,20 @@ def __init__(self, data, times, comment=""):
         times : array, double
             Times in seconds at which the data is sampled. The number of times
             must match the `size` of the last dimension of `data`.
-        comment : str
-            A comment related to `data`. The default is ``''``, which
+        comment : str, optional
+            A comment related to `data`. The default is ``""``, which
             initializes an empty string.
+        complex : bool
+            A flag which indicates if the raw data are real or complex-valued.
+            The default is ``False``.
 
         """
 
         _Audio.__init__(self, 'time', comment)
 
+        assert isinstance(complex, bool)
```

Comment:
I think this should raise a more verbose error

``` 
if not isintance(...):
    raise TypeError(f"complex is {complex} but must be True or False")
```

Reply:
see above